.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
     - PIP_CACHE_DIR="$HOME/.cache/pip"  # unify pip cache location for all platforms
 stages:
 #  - check
-#  - test
+  - test
   - build
```

Comment:
I suppose we don't support publishing yet - it should be failing, no?

Reply:
apparently, @efiop have setup publishing already https://github.com/iterative/PyDrive2/commit/d44c23c7ed9df424e9199ae74bcc48a51eccaf70 to run the corresponding script https://github.com/iterative/PyDrive2/blob/master/scripts/build_package.sh#L12